run_api.py

```diff
@@ -23,5 +24,5 @@
     if len(sys.argv) > 1 and sys.argv[1].lower().startswith('test'):
         doctest.testmod()
     else:
-        debug = not os.getenv('PRODUCTION')
+        debug = not ast.literal_eval(os.getenv('PRODUCTION', False))
```

Comment:
Looks like this will fail if the `PRODUCTION` environment variable isn't set: `ast.literal_eval(False)` raises a `ValueError`. Could change `False` to `'False'` to fix.